### DISCUSSION TITLE
Sunbird ED installation on Ubuntu 20.04/21.04

### DISCUSSION THREAD

**Discussion by mahesh-maney** [Sunbird-Ed/Community#582] [Installation] [answered]
Hi,
Since Ubuntu 18.04 has been decommissioned, we need to install sunbird ED on Ubuntu 20.04. 
What are the implications of;
-        Sunbird-ED and Sunbird-CoKreat installations
-        Build and Deploy of services
Has anyone attempted to install  Sunbird-ED and Sunbird-CoKreat on Ubuntu 20.04.

**Comment by amitpriyadarshi** [comment]
@santhosh-tg @tsprasath can you provide any link for asked details.
cc: @gandham-santhosh @Krishnaj20 @vinukumar-vs

**Reply by gandham-santhosh** [reply, marked as answer]
@mahesh-maney We are aware that recent Diksha setup was done on Ubuntu 20.04 LTS. If you face any challenges with npm packages or python packages not being available, you can fork and update the changes accordingly and contribute back

**Reply by mahesh-maney** [reply]
Thank you @gandham-santhosh
